app/assets/stylesheets/application.css

```
@@ -1,7 +1,7 @@
 @import "tailwindcss" source(none);
 @source "../../views/**/*.slim";
 @source "../../components/**/*.slim";
-@source "../../../app/javascript/**/*.{js,vue,jsx}";
+@source "../../../app/javascript/**/*.{js,vue}";
```

Stylelint (17.11.1)

[error] Unexpected unknown at-rule "`@source`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)